### PR TITLE
stalled tunnel detection changes

### LINKED ISSUE #760: stalled tunnel detector too fast
On startup, 1000's of services on some machines takes too long and triggers the stalled tunnel detector

Make it configurable and increase the default

### PR DESCRIPTION
**Pull request by dovholuknf**
* changes the stall detection from 3 iterations (~15s) to 12 iterations (~60s)
* allows the user to change this setting if they are an administrator (no ipc support)

closes #760

### CODE REVIEW

File: ZitiUpdateService/utils/Settings.cs [hunk 1]
```diff
@@ -77,8 +79,8 @@ internal void Load() {
                     Logger.Debug("settings file was null? file doesn't exist or file was garbage?");
                 }
             } catch (Exception ex) {
-                // do nothing, probably means the file is just or doesn't exist etc.
-                Logger.Debug("unexpected error loading settings. file was null? file doesn't exist or file was garbage? {0}", ex);
+                // probably means the file is doesn't exist or hasn't finished being written yet
```
Comment (scareything) [inline ZitiUpdateService/utils/Settings.cs:82]: "the file is doesn't..."

Comment (dovholuknf) [reply ZitiUpdateService/utils/Settings.cs:82]: thx

File: ZitiUpdateService/utils/Settings.cs [hunk 2]
```diff
@@ -116,14 +118,37 @@ private void OnCreated(object sender, FileSystemEventArgs e) {
...
+                    try {
+                        Load();
+                        //this.OnConfigurationChange?.Invoke(null, null);
+                    } catch (Exception ex2) {
+                        Logger.Debug("unexpected error loading settings twice. not trying again. {0}", ex2);
+                    }
+                } else {
+                    Logger.Debug("unexpected error loading settings. file was null? file doesn't exist or file was garbage? {0}", ioe);
+                }
+                this.OnConfigurationChange?.Invoke(null, null);
```
Comment (scareything) [inline ZitiUpdateService/utils/Settings.cs:142]: is my understanding correct that this only happens when there was an io exception when loading?

Comment (dovholuknf) [reply ZitiUpdateService/utils/Settings.cs:142]: oh man nice catch. thanks. that was supposed to be inside the previous catch

**Pull request closed without merging**